<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>principal</class>
 <widget class="QMainWindow" name="principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>MS UI Gothic</family>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>PET-EATER</string>
  </property>
  <property name="windowIcon">
   <iconset theme="PET">
    <normaloff>:/icono/PET.png</normaloff>:/icono/PET.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="fondo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	
	
	background-color: rgb(212, 236, 212);
	border-radius:20px;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="ajustes">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>130</y>
       <width>171</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/imagenes/informacion.png);
background-color: rgb(168, 186, 168);
border-radius:10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QProgressBar" name="nivelalmacenamiento">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>460</y>
       <width>251</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar{
	border-style:none;
	text-align:center;
	font: 30pt &quot;MS Shell Dlg 2&quot;;
}

QProgressBar::chunk{
	
	
	background-color: qlineargradient(spread:pad, x1:0, y1:0.528, x2:1, y2:0.523, stop:0 rgba(99, 110, 99, 255), stop:1 rgba(144, 240, 144, 255));
}</string>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
    <widget class="QLabel" name="etiquetallenado">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>420</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; font-style:italic;&quot;&gt;Nivel de llenado&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="estadisticas">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>420</y>
       <width>171</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/imagenes/estadis.png);
background-color: rgb(168, 186, 168);
border-radius:10px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>60</y>
       <width>461</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>21</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt; font-weight:600; font-style:italic;&quot;&gt;PET-EATER V1.0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="iniciar">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>160</y>
       <width>531</width>
       <height>281</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>30</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(144, 240, 144);

gridline-color: rgb(255, 255, 255);

border-top-color: rgb(117, 255, 93);
font: 75 30pt &quot;Century Gothic&quot;;

border-radius:30px;
</string>
     </property>
     <property name="text">
      <string>Comienza a Reciclar</string>
     </property>
    </widget>
    <widget class="QLabel" name="notificacion">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>450</y>
       <width>511</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic </family>
       <pointsize>26</pointsize>
       <weight>5</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);
font: 45 26pt &quot;Century Gothic &quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>200</y>
       <width>91</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
image: url(:/imagenes/botella2.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="etiquetallenado_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>261</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; font-style:italic;&quot;&gt;¿Usuario nuevo?&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt; font-weight:600; font-style:italic;&quot;&gt;Presióname &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="etiquetallenado_3">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>370</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Century Gothic</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:600; font-style:italic;&quot;&gt;Datos&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLCDNumber" name="Hora">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>100</y>
       <width>201</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="fecha">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>10</y>
       <width>201</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="imag.qrc"/>
 </resources>
 <connections/>
</ui>
